src/Discord.Net.Commands/Discord.Net.Commands.csproj

```diff
@@ -6,6 +6,9 @@
     <Description>A Discord.Net extension adding support for bot commands.</Description>
     <TargetFrameworks>netstandard1.1</TargetFrameworks>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard1.1|AnyCPU'">
```

Comment:
Why do we need C# 7.3?

Reply:
Leftover from me fiddling with ValueTuples before deciding against them, forgot about this.